server/src/main/java/org/eclipse/lsp/cobol/core/strategy/ErrorMessageUtil.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package org.eclipse.lsp.cobol.core.strategy;
+
+import com.google.common.collect.ImmutableSet;
+import org.antlr.v4.runtime.InputMismatchException;
+import org.antlr.v4.runtime.NoViableAltException;
+import org.antlr.v4.runtime.Parser;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.misc.IntervalSet;
+import org.eclipse.lsp.cobol.core.CobolParser;
+import org.eclipse.lsp.cobol.core.messages.MessageService;
+
+import java.util.*;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toList;
+import static org.antlr.v4.runtime.Recognizer.EOF;
+
+/** This is a utility class to fetch error messages encountered while parsing COBOL file. */
+public class ErrorMessageUtil {
+  private final MessageService messageService;
+  static final String PERFORM_MISSING_END = "ErrorStrategy.performMissingEnd";
+  static final String REPORT_UNWANTED_TOKEN = "ErrorStrategy.reportUnwantedToken";
+  static final String END_OF_FILE_MESSAGE = "ErrorStrategy.endOfFile";
+  static final String REPORT_INPUT_MISMATCH = "ErrorStrategy.reportInputMismatch";
+  private static final String MSG_DELIMITER = ", ";
+  private static final String MSG_PREFIX = "{";
+  private static final String MSG_SUFFIX = "}";
+  private static final Map<Class<? extends Parser>, Set<String>> IDENTIFIER_TOKENS =
+      IdentifierReplacing.retrieveTokenToRemove();
+
+  ErrorMessageUtil(MessageService messageService) {
```

Comment:
Sorry, but I disagree. A utility class can't be instantiated. Can we rename it to ErrorMessageHelper or Service, or whatever?

Reply:
Yes, I agree. Updated

Reply:
Thanks)